app/src/main/java/com/eveningoutpost/dexdrip/services/G5BaseService.java

```diff
@@ -197,4 +197,4 @@ public static String getLastTwoCharacters(final String txid) {
         if (txid == null) return "NULL";
         return txid.length() > 3 ? txid.substring(txid.length() - 2) : "ERR-" + txid;
     }
-}
+}
```

Comment:
add new line

Reply:
I didn't remove the last line.  It was like that.

Reply:
Done